modules/govuk/manifests/node/s_backend.pp

```diff
@@ -28,6 +28,8 @@
     ensure => installed,
   }
 
+  include govuk_aws_xray_daemon
```

Comment:
If we're going to have config like this which applies to all app nodes, does it make sense to have an `s_app_base` class?  There are a few other classes like this.

Reply:
Yeah, we have https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/node/s_app_server.pp but it has the comment "This class is deprecated; nothing new should be added to it and things should be removed over time."